WordPress/Classes/Services/WordPressComSyncService.swift

```diff
@@ -48,12 +48,12 @@ class WordPressComSyncService {
             onSuccess(account)
         }
 
-        if isJetpackLogin && !accountService.isDefaultWordPressComAccount(account) {
+        if isJetpackLogin && !account.isDefaultWordPressComAccount {
             let blogService = BlogService(managedObjectContext: context)
             blogService.associateSyncedBlogs(toJetpackAccount: account, success: onSuccessInternal, failure: onFailureInternal)
 
         } else {
-            if accountService.defaultWordPressComAccount()?.isEqual(account) == false {
+            if !account.isDefaultWordPressComAccount {
```

Comment:
No strong opinion on it, though the main improvement here is IMHO the fact that we don't need an `AccountService` object to tell us whether a given account is the default WP.com one. 

I'd be fine to rework this – tbh I tend to try to avoid `!thing` as well, but usually via restructuring the surrounding code using `guard` or something.

For now, I'd leave this since we'll be coming back to it anyway in the future, but that's just me